core/foundation/res/ROOT/FoundationUtils.hxx

```diff
@@ -0,0 +1,37 @@
+/// \file FoundationUtils.hxx
+///
+/// \brief The file contains utilities which are foundational and could be
+/// across the core component of ROOT.
+///
+///
+/// \author Vassil Vassilev <vvasilev@cern.ch>
+///
+/// \date June, 2019
+///
+/*************************************************************************
+ * Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_CORE_FOUNDATION_FOUNDATIONUTILS_HXX
+#define ROOT_CORE_FOUNDATION_FOUNDATIONUTILS_HXX
+
+#include <string>
+
+namespace ROOT {
+namespace FoundationUtils {
```

Comment:
Maybe `PathUtils.hxx` given its current content?

Reply:
I intended to be used for more general utilities. I’d prefer the namespace name to be more generic as it’d be more difficult to change it later.